docling/models/tesseract_ocr_model.py

```
@@ -57,6 +57,22 @@ def __init__(self, enabled: bool, options: TesseractOcrOptions):
             _log.debug("Initializing TesserOCR: %s", tesseract_version)
             lang = "+".join(self.options.lang)
 
+            if lang != "auto":
+                for lg in self.options.lang:
+                    if lg in ("jpn", "jpn_vert"):
+                        # Check if the language or its script prefixed version is available
+                        lang_available = lg in self._tesserocr_languages
+                        if self.script_prefix:
+                            lang_available = lang_available or f"{self.script_prefix}{lg}" in self._tesserocr_languages
```

Pylint (3.3.7)

[convention] Line too long (119/100)

(C0301)